### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by jxGui** [coreemu/core#755] [General]
Description:
I want to emulate an ad-hoc network on CORE . I used the method about building ad-hoc network on Linux. However, due to there is no wireless network interface card in node like MDR\PC and so on, when I run the command: sudo iwconfig lo mode ad-hoc, it shows Error (figure1). Do you know another way to build ad-hoc network in CORE. I will really appreciate it if you can tell me. 

screenshot:
<img width="540" alt="5aaeef9c8b7754b38a7cb40863ab465" src="https://user-images.githubusercontent.com/101703577/218800922-f33a0f3d-b882-43d9-b06b-98d2d8e2fac9.png">

**Comment by bharnden** [comment]
You are still speaking in generalities without a specific purpose, an ad-hoc network can be a very open ended concept.

Specifically iwconfig is not used currently. If you wanted to use it, you would need to execute everything related to that on your own.

If you want to run networks that are similar in nature, you can use the WLAN, Wireless, or EMANE nodes. All provide node connectivity to help facilitate ad-hoc networks.

See here: https://coreemu.github.io/core/gui.html#wireless-networks
